Hunting Queries/SecurityEvent/ServiceInstallationFromUsersWritableDirectory.yaml

```diff
@@ -0,0 +1,38 @@
+id: 5a9ccb48-1316-46e1-89d1-aca0355c305e
+name: Service installation from user writable directory
+description: |
+   'This query detects a service installation that is originated from a user writable directory.
+   Ref: https://attack.mitre.org/techniques/T1569/002/'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+tactics:
+  - Execution
+relevantTechniques:
+  - T1569
```

Comment:
You can include the sub-technique here (T1569.002)

Reply:
Fixed